tools/asciidoctor/cli.adoc

```diff
@@ -0,0 +1,197 @@
+// Copyright (C) 2020  Matthew Glazar
+// See end of file for extended copyright information.
+
+= quick-lint-js(1)
+:doctype: manpage
+:version: 0.2.0
+:manmanual: General Commands Manual
+:mansource: quick-lint-js version {version}
+:toc:
+:reproducible:
+:stylesheet: main.css
+:linkcss:
+
+== Name
+
+quick-lint-js - find bugs in JavaScript programs
+
+== Synopsis
+
+*quick-lint-js* [*--output-format*=_format_] [<options>] _file_ [_file_...] +
+*quick-lint-js* *--lsp-server* [<options>]
+
+== Description
+
+*quick-lint-js* reads JavaScript files and reports syntax errors and other bugs.
+
+This command has two modes:
+
+*quick-lint-js* [<options>] _file_ [_file_...]::
+  Batch mode (default).
+  Check the given files, and report errors to the terminal (standard error).
+  --output-format can be used to customize how errors look.
+
+*quick-lint-js* --lsp-server::
+  LSP server mode.
+  Start a Language Server Protocol server, communicating using standard input and standard output.
+  Use this mode to integrate with code editors supporting LSP.
+
+== Options
+
+*--output-format*=_format_::
+  Customize how errors are printed. _format_ is one of the following:
++
+pass:[-] *gnu-like* (default): a human-readable format similar to GCC.
++
+pass:[-] *vim-qflist-json*: machine-readable JSON which can be given to Vim's _setqflist_ function.
++
+Incompatible with *--lsp-server*.
+
+*--vim-file-bufnr*=_number_::
+  Set the _bufnr_ property for errors printed with the *--output-format=vim-qflist-json* option.
+  *--vim-file-bufnr* applies only to files which are given later in the command line.
+  Therefore, if multiple files are given, *--vim-file-bufnr* can be specified multiple times.
+
+*--exit-fail-on*=_errors_::
+  Cause *quick-lint-js* to exit with a non-zero exit code if any of the discovered errors is listed in _errors_.
++
+See the "ERROR LISTS" section for a description of the format for _errors_.
++
+Incompatible with *--lsp-server*.
+
+*--lsp, --lsp-server*::
```

Comment:
Should we put these on separate lines like line 139-140?
```suggestion
*--lsp*::
**--lsp-server*::
```

Reply:
For the HTML this would leave us with:

```
--lsp
--lsp-server
    blah...
```

It's weird behavior between the HTML backend and the man page backend...

Reply:
> For the HTML this would leave us with:
> 
> ```
> --lsp
> --lsp-server
>     blah...
> ```

That looks fine to me.